[IMP] referrals: update pics and remove save button (#8686)
odoo/documentation: content/applications/hr/referrals.rst

Odoo's *Referrals* application is a centralized place where all information regarding referrals is
housed, from points earned, coworkers hired, and rewards selected. Users can recommend people they
know for job positions, and then earn referral points as those people progress through the
recruitment pipeline. Once enough referral points are earned, they can be exchanged for prizes. The
*Referrals* application integrates with the *Employees*, *Recruitment*, and *Website* applications,
all of which must be installed in order for the *Referrals* application to function.

The only configurations needed for the Referrals application after it is installed are the
:ref:`rewards <referrals/rewards>`; everything else is pre-configured when Odoo Referrals is
installed.

Any user with either :guilabel:`Referral User`, :guilabel:`Officer`, or :guilabel:`Administrator`
rights for the Recruitment application has access to the Referrals application. Only users with

> :guilabel:`Administrator` rights for the Recruitment application have access to the :ref:`reporting

Reviewer: ```suggestion
:guilabel:`Administrator` access rights for the *Recruitment* application have access to the :ref:`reporting
```

> the Recruitment application can modify onboarding slides. To edit a slide, navigate to

Reviewer: ```suggestion
the *Recruitment* application can modify onboarding slides. To edit a slide, navigate to
```

Reviewer: Not sure if the sentence: "To edit a slide, navigate to the Recruitment application can modify onboarding slides."
Can you please rework this?

Author: This is the full text in this section:

To edit a slide, navigate to :menuselection:`Referrals app --> Configuration --> Onboarding.` Each line displays the text for the individual onboarding slide. To edit an onboarding slide, click on an individual slide line to open the slide's onboarding form.

I don't see what is missing?

:menuselection:`Referrals app --> Configuration --> Onboarding.` Each line displays the text for the
individual onboarding slide. To edit an onboarding slide, click on an individual slide line to open
the slide's onboarding form.

Make any changes to the message in the :guilabel:`Text` field. A :guilabel:`Company` may be

> selected, however, if this field is populated, that slide will *only* be displayed for that

Reviewer: ```suggestion
selected, however, if this field is populated, that slide is *only* displayed for that
```

> selected, however, if this field is populated, that slide will only be displayed for that particular

Reviewer 2: ```suggestion
selected, however, if this field is populated, that slide will *only* be displayed for that particular
```
Optional for emphasis

company.

.. note::

>    The :guilabel:`Company` field only appears when in a multi-company database.

Reviewer 2: ```suggestion
   The :guilabel:`Company` field **only** appears when in a multi-company database.
```

> The image can be modified as well. Hover over the image thumbnail in the top-right corner of the

Reviewer: ```suggestion
The image can be modified, as well. Hover over the image thumbnail in the top-right corner of the
```

form. A :guilabel:`✏️ (pencil)` icon and :guilabel:`🗑️ (garbage can)` icon appear. Click the
:guilabel:`✏️ (pencil)` icon to change the image. A file navigator window loads. Navigate to the
desired image, select it, then click :guilabel:`Open`. The new image appears in the thumbnail. To
delete an image, click the :guilabel:`🗑️ (garbage can)` icon, then select a new image using the
:guilabel:`✏️ (pencil)` icon.

.. image:: referrals/edit-onboarding.png
   :align: center
   :alt: An onboarding slide in edit mode, with the main fields highlighted.

The slide can be saved manually at any time by clicking the *save manually* option, represented by a
:guilabel:`(cloud upload)` icon, located at the end of the slide description in the top-left corner.
To cancel any changes made, click the :guilabel:`↺ (Discard changes)` icon to delete any changes and

> revert to the original content.

Reviewer: I don't think this section is necessary, as this function appears on nearly each page in the database. So, I think you can delete this.

> The sequence the slides appear can be changed from the Onboarding dashboard. Click the up and down

Reviewer 2: ```suggestion
The sequence in which the slides appear can be changed from the Onboarding dashboard. Click the up and down
```
Suggestion:
If we're on the onboarding dashboard: change to :guilabel:`Onboarding dashboard`
or, if we're not on the dashboard,
change to: *Onboarding* dashboard

Author: Good suggestion- we WERE but then I had people jump to the actual individual slide form. So italics itis!

and down arrow symbol that appears after the checkbox and before the slide text, and drag the slide

> to the desired position.

Reviewer: I'm not seeing an up/down arrow -- just the typical "six dot" icon that needs to be clicked to activate the drag-and-drop...can you please double-check this during your revisions?

Author: Good catch! I originally was only updating images, then added some more changes, so I missed this!

Share job positions
===================

In order to earn referral points, a user must first share a job position, so the applicant can apply
for the position. Job positions can be shared in several ways within the *Referrals* application,
through the :ref:`View Jobs <referrals/view-jobs>` button and the :ref:`Email A Friend

> <referrals/email-jobs>` button, located at the bottom of the *Referrals* app dashboard.

Author: Good idea- adding  note block

===================

In order to earn referral points, a user must first share a job position so the applicant can apply

> for the position. Job positions can be shared in several ways within the Referrals application,

Reviewer 2: ```suggestion
for the position. Job positions can be shared in several ways within the *Referrals* application,
```

through the :ref:`View Jobs <referrals/view-jobs>` button and the :ref:`Email A Friend

> <referrals/email-jobs>` button.

Reviewer: where in the application are these buttons found? a little specificity here would be helpful

Share a job
-----------

> Other than sending an email, job positions can be shared via social media platforms, and tracking

Reviewer: ```suggestion
Other than sending an email, job positions can be shared, via social media platforms, and tracking
```

> links to the job position. At the bottom of each job position card are four icons and corresponding
> tracking links that can be used to share the job position, keeping track of applicants in the
> *Referrals* application.

Reviewer: ```suggestion
links to the job position. At the bottom of each job position card are four icons, and corresponding
tracking links, that can be used to share the job position, keeping track of applicants in the
*Referrals* application.
```

Share a job
-----------

> Other than sending an email, job positions can be shared in several other ways, via social media
> platforms, and tracking links to the job position. At the bottom of each job position card are four

Reviewer 2: ```suggestion
Other than sending an email, job positions can be shared via social media
platforms, and tracking links to the job position. At the bottom of each job position card are four
```

To share the job position with a customized tracking link, click the :guilabel:`Share Now` button
with the :guilabel:`🔗 (chain)` icon inside it. A :guilabel:`Link to Share` pop-up window appears
with the tracking link. Click :guilabel:`Copy` to copy the link. After the link is copied, click the
:guilabel:`Close` button to close the pop-up window. Next, share the link with the prospective
employee however desired.

Facebook
~~~~~~~~

To share the job position using Facebook, click the :guilabel:`Share Now` button with the
:guilabel:`f (Facebook icon)` inside it.

If the user is already logged into Facebook, when the :guilabel:`f (Facebook icon)` is clicked, a
:guilabel:`Share on Facebook` page loads in a new tab, with the link populated in the main body of
the new post in a pop-up window. If the user is *not* already logged in, a log-in screen loads,
instead, prompting the user to log-in to Facebook first.

Type in any additional information to add to the post, then share the job position using the
available options in Facebook.

X/Twitter
~~~~~~~~~

> A job position can also be shared on X/Twitter. Click the :guilabel:`Share Now` button with the bird

Author: OMG! Iput a ticket in and it was going to be done at some point, but I was not aware it already happened! Thank you- I am glad they changed that (I'll update my image now, too!)

To share the job position with a customized tracking link, click the :guilabel:`Share Now` button
with the :guilabel:`🔗 (chain)` icon inside it. A :guilabel:`Link to Share` pop-up window appears
with the tracking link. Click :guilabel:`Copy` to copy the link. After the link is copied, click the
:guilabel:`Close` button to close the pop-up window. Next, share the link with the prospective
employee.

Facebook
~~~~~~~~

To share the job position using Facebook, click the :guilabel:`Share Now` button with the

> :guilabel:`f (Facebook icon)` inside it.

Reviewer 3: ```suggestion
:guilabel:`f (Facebook)` icon inside it.
```

with the link populated in the main body of the new post. If the user is *not* already logged in, a

> log-in screen loads instead, prompting the user to log-in to Facebook first.

Reviewer: ```suggestion
log-in screen loads, instead, prompting the user to log-in to Facebook first.
```

> Type in any additional information to add to the post, in the space directly below the user's name.

Reviewer: ```suggestion
Type in any additional information to add to the post in the space directly below the user's name.
```

If desired, :guilabel:`tag` users, :guilabel:`check-in` at a location, or add a :guilabel:`status`

> to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag to
> the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location

Reviewer 2: ```suggestion
to the post. To tag a user, click the :guilabel:`Tag Friends` icon and select the people to tag in
the post. To add a check-in location, click the :guilabel:`Check in` icon, and select the location
```

from the pop-up of predetermined locations based on the user's current location. To add a

> :guilabel:`status` click the :guilabel:`Feeling/Activity` icon, then select either the feeling or

Reviewer 2: ```suggestion
:guilabel:`status`, click the :guilabel:`Feeling/Activity` icon, and select either the feeling or
```

> the activity type, and lastly, if an activity type was selected, select the specific activity.

Reviewer: I don't believe 'tag', 'check-in', nor 'status' need a GUI, since that is not how those fields appear on the UI

Can you also break up this large paragraph, please?

Describe the check in icon

mention "window" (or "form" when applicable) after every mention of "pop-up"

can you describe the look of the 'feeling/activity' icon?

> Five methods are available to share the job position, :guilabel:`Your Story`, :guilabel:`Share on a

Reviewer 2: ```suggestion
Five methods are available to share the job position: :guilabel:`Your Story`, :guilabel:`Share on a
```

      At the lower portion of the form are radio buttons next to the two options, :guilabel:`Feed`
      (for the News Feed) and :guilabel:`Your Story` for stories. When :guilabel:`Share to News Feed
      or Story` is selected, these buttons are active, if another option is selected, these radio
      buttons are grayed out. Activate the radio buttons to indicate how the job position should be

>       shared, either in the news feed, the story, or both.

Reviewer 2: ```suggestion
      shared: either in the news feed, the story, or both.
```

Click the :guilabel:`Email a friend` button at the bottom of the screen. A :guilabel:`Send Job Offer
by Mail` pop-up window appears.

Enter the email address in the :guilabel:`Email` line. The email can be sent to multiple
recipients, simply enter a comma followed by a single space after each email address. The
:guilabel:`Subject` is pre-configured with `Job for you`, but can be edited.

The email body is also populated with pre-configured text. The text that appears is:

`Hello,`

`There are some amazing job offers in my company! Have a look, they can be interesting for you\:`

> `(Company)/Job Offers`

Author: I thnk this was either because of multi-company, or just how it looked in previous versions. Updating!

:guilabel:`Amazing job offer for (Job Position)! Check it live: (link to Job Position)`

Type in any additional information or make any edits to the message. If desired, click the
:guilabel:`X` in the top-left corner of the thumbnail preview of the webpage to remove it. To adjust
the audience for the message, click the drop-down button labeled :guilabel:`Everyone`. Select the
preferred audience from the drop-down menu. The default option is :guilabel:`Everyone`, and the
other option is :guilabel:`Circle`. The :guilabel:`Circle` is a group of people that the user has
selected in X/Twitter.

The default setting is :guilabel:`🌎 (globe) Everyone can reply`, an allows everyone on X/Twitter to
reply to the message. To change this, click the :guilabel:`🌎 (globe) Everyone can reply` button in
the lower left section of the message. Four options are presented: :guilabel:`Everyone`,
:guilabel:`Accounts you follow`, :guilabel:`Verified accounts`, and :guilabel:`Only accounts you
mention`. Selecting one of the other options restricts who can reply to the message.

> Various other items that can be added to the message. Media (photos, videos, etc), GIFs, polls,

Reviewer 2: ```suggestion
Various other items that can be added to the message. Media (photos, videos, etc.), GIFs, polls,
```

:guilabel:`🗑️ (garbage can)` icon. Click the :guilabel:`✏️ (pencil)` icon and a file explorer
window appears. Navigate to the desired image file then click :guilabel:`Open` to select it.

The referral friend form automatically saves but can be saved manually at any time by clicking the
*save manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left

> corner. To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete

Reviewer: is it an 'X' or a 'counter-clockwise arrow'?

Author: Still an X- I checked in the latest (Runbot 17.2)

:guilabel:`My Referral` window loads. This screen displays all the referrals, with each referral in
its own referral card.

A successful referral displays a green :guilabel:`Hired` banner in the top-right corner of the card.
Referrals that are in process display a purple :guilabel:`In Progress` banner in the top-right
corner.

> Each referral card lists the name of the applicant, the subject/application title as it appears on
> the applicant card in the *Recruitment* app, the name of the job position, the person responsible
> for filling the role (recruiter), and the points earned. For referrals that have been hired, the
> selected avatar also appears on the card.

Reviewer: ```suggestion
Each referral card lists the name of the applicant, the subject/application title (as it appears on
the applicant card in the *Recruitment* app), the name of the job position, the person responsible
for filling the role (recruiter), and the points earned. For referrals that have been hired, the
selected avatar also appears on the card.
```

Instead of making a public post, the post can be sent to a connection in a private message. To do
so, click the :guilabel:`Send as private message` button in the lower left corner. A message window
loads with the link to the job description in the bottom of the window.

First, select the recipients for the message. Type the name in the :guilabel:`Search for message
recipients` field. As the name is typed in, any matching connections appear. Click on a connection
to add them. Repeat this for all the recipients to send the message to. There is no limit on the
number of connections that can be messaged.

Next, enter any additional text in the message body. Click :guilabel:`Send` in the lower-right

> corner to send the message. A :guilabel:`Message successfully sent!` screen loads. Two options are

Reviewer 2: ```suggestion
corner to send the message. Once sent, a :guilabel:`Message successfully sent!` screen loads. Two options are
```

   - :guilabel:`Contract Signed`: 50 points

   The total points earned for a referral that is hired is 85 points. Modifications of the points

>    earned for each stage are done in the Recruitment application. Refer to the

Reviewer: ```suggestion
   earned for each stage are done in the *Recruitment* application. Refer to the
```

> The rewards are the only configurations needed when setting up the Referrals application. Only users

Reviewer: ```suggestion
The rewards are the only configurations needed when setting up the *Referrals* application. Only users
```
again, please go through this PR to ensure all application titles are formatted correctly. I will stop doing them personally from here on out --  but please double-check their accuracy during your revisions. thanks!

with :guilabel:`Administrator` rights for the Recruitment application can create or modify rewards.
To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click

> :guilabel:`New` and a reward form loads. Enter the following information on the form:

Reviewer: ```suggestion
:guilabel:`New`, and a reward form loads. Enter the following information on the form:
```

> - :guilabel:`Product Name`: Enter the name as it should appear for the reward.

Reviewer: please make sure that all words that immediately follow a ":" are lower-case.
```suggestion
- :guilabel:`Product Name`: enter the name as it should appear for the reward.
```
just fixing this one -- ensure similar formatting follows suit throughout the PR. thanks!

- :guilabel:`Cost`: Enter the amount of points required to redeem the reward.

> - :guilabel:`Company`: Using the drop-down menu, select the company the reward will appear for. If a

Reviewer: ```suggestion
- :guilabel:`Company`: Using the drop-down menu, select the company the reward appears for. If a
```

  reward is to be used for several companies, each company needs to have a record of the reward,
  with the company listed on the form. This field only appears if in a multi-company environment.
- :guilabel:`Gift Responsible`: Select the person who will be responsible for procuring and
  delivering the reward to the recipient, using the drop-down menu. This person is alerted when the

>   reward is purchased, so they know when to deliver the reward to the recipient.

Reviewer: just to clarify...are these rewards "purchased" or "earned" via points received?

Author: ah, I see - it's earned using the rewards- you have to "spend" the points you earned.

Author: I changed it to "boughtin the *Referrals* app" sinc ethe word in the app is "buy" so the nomenclature matches but it is specific to being "bought" IN the app- so I think that clarifies it.

Click the :guilabel:`Email a friend` button at the bottom of the screen. A :guilabel:`Send Job Offer
by Mail` pop-up window appears.

Enter the email address in the :guilabel:`Email` line. The email can be sent to multiple
recipients, simply enter a comma followed by a single space after each email address. The
:guilabel:`Subject` is pre-configured with `Job for you`, but can be edited.

The email body is also populated with pre-configured text. The text that appears is:

`Hello,`

`There are some amazing job offers in my company! Have a look, they can be interesting for you\:`

`(Company)/Job Offers`

The `(Company)/Job Offers` text is a tracking link to a complete list of all job positions currently
being recruited for, that are live on the company's website (published). Add any additional text

> and make any edits to the message body, then click :guilabel:`Send Mail` to send the email. The

Reviewer 2: ```suggestion
and make any edits to the message body, if necessary. Then, click :guilabel:`Send Mail` to send the email. This
```

> message will be sent and the window will close.

Reviewer 2: ```suggestion
sends the message and closes the window.
```

>   Click the :guilabel:`✏️ (pencil)` icon and a file explorer appears. Navigate to the photo file,

Reviewer: ```suggestion
  Click the :guilabel:`✏️ (pencil)` icon, and a file explorer appears. Navigate to the photo file,
```

shields.

To modify a level's image, name, or points required to reach the level, click on an individual level
in the list to open the level form, then make modifications.

> Type in the name or number of the level in the :guilabel:`Level Name` field. What is entered is

Reviewer: ```suggestion
Type in the name (or number) of the level in the :guilabel:`Level Name` field. What is entered is
```

> (pencil)` icon and a file explorer window appears. Navigate to the desired image file then click
> :guilabel:`Open` to select it.

Reviewer: ```suggestion
(pencil)` icon, and a file explorer window appears. Navigate to the desired image file, then click
:guilabel:`Open` to select it.
```

> The level form saves automatically but can be saved manually at any time by clicking the *save

Reviewer: ```suggestion
The level form saves automatically, but can be saved manually at any time by clicking the *Save
```

manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left corner.

> To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete any

Reviewer: is it an 'X' or a 'counter-clockwise arrow'?

Author: Yup- it's an X

> changes and revert to the original content.

Reviewer: ```suggestion
changes, and revert to the original content.
```

  the alert should be displayed for, select the company from the drop-down menu. If this field
  remains blank, the alert is visible to everyone with access to the Referrals application. If a
  company is specified, only user's within that company (who also have access to the Referrals

>   application) will see the alert. This field only appears when in a multi-company environment.

Reviewer 2: ```suggestion
  application) see the alert. This field **only** appears when in a multi-company database.
```
Changed environment to database to keep consistent with how it was written earlier - but choose which one you like better :-)
